### ISSUE TITLE
[BUG] Hidden components still have a margin-bottom, causing vertical misalignments in columns

### ISSUE THREAD

**Issue by franke-mc** [formio/formio.js#1960]
## Environment
- Hosting type
  - [x] Form.io
- Formio.js 4.5.0
- Browser: firefox 70.0

## Example form on next.form.io:
- [see here](https://api.form.io/project/5d1c9ef99cf71913daa0e494/form/5db3030076ca20ddaa8b10ac/)

## Steps to Reproduce
![hiddenMarginBottom edit](https://user-images.githubusercontent.com/46748541/67624548-0ce68d00-f832-11e9-9367-0062256a8d2a.png)
1. Create a columns component with two columns
2. Put a text field in the right column
3. Put two text fields in the left column, one below the other
4. Make the upper left text field hidden.
5. Use the form to enter data

## Expected behavior
In the left column, the second text field moves up to take the place of the (hidden) first field, to align with the field in the right column.

## Observed behavior
The `<div>` of the (hidden) first field in the left column still exists with its margin-bottom, causing the second field on the left to not be aligned.

![hiddenMarginBottom use](https://user-images.githubusercontent.com/46748541/67624550-1112aa80-f832-11e9-91d5-e056b355ec14.png)

**Comment by jhen1422** [comment]
@franke-mc 
Thank you for the report. We were able to replicate this issue on our end as well and have created a ticket in our 4x branch. We will review during our planning meeting next week. Please note, we welcome pull requests in this library. If you need this item expedited, please feel free to contact support@form.io for pricing.

**Comment by franke-mc** [comment]
Thanks @randallknutson for 2d1e7cdf85865f9917c5629618071f272a48f211 (bugfix/hidden-margin2).
I can confirm that this works in our application as long as I add another css rule:
```
.form-group.formio-hidden {
  margin-bottom: 0;
}
```
to override the rule
```
.form-group {
  margin-bottom: 1rem;
}
```
from `node_modules/bootswatch/dist/*/bootstrap.css`. Not sure about the details of how all those stylesheets get combined, though, so it may or may not be necessary to add this to formio.js, too.

**Comment by wag110894** [comment]
Hello @franke-mc,

This issue has been resolved in version 4.8.0-rc3. 

Thank you for using Form.io and please let us know if you have any further questions.

William Gay
